<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>band_pass_settings</class>
 <widget class="QWidget" name="band_pass_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>band_pass_filter</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_band_pass_settings_enabled">
     <property name="toolTip">
      <string>Fourier Bandpass</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="slim5.qrc">
       <normaloff>:/images/black.svg</normaloff>:/images/black.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="band_filter_min_dx">
     <property name="toolTip">
      <string>Smallest Size (Pixels)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="band_filter_max_dx">
     <property name="toolTip">
      <string>Largest Size (Pixels)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="band_filter_remove_dc">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Keep DC</string>
     </property>
     <property name="text">
      <string>DC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
